tests/scripts/remote_monitoring_tests/Remote_monitoring_tests.md

```diff
@@ -44,6 +44,21 @@ Here are the test scenarios:
 - Test with invalid values such as blank, null or an invalid value for various keys in the updateResults json
 - Update the same results twice for the same experiment
 
+Namespace Related Test Scenarios:
+Sanity Tests
+- Update results for a single valid namespace experiment
+- Update multiple valid results in a single json for a single namespace experiment
+- Update multiple valid results for a namespace experiment (uses a loop)
+- Update results test with supported cpu and memory format types
+- Update the same results twice for the same namespace experiment
+- Update results with valid invalid interval duration
+
+Negative Tests
+- Update results for namespaces that are not present during creation of the namespace experiment [Fail]
```

Comment:
Are these implemented?

Reply:
Are you raising a separate PR for this? Can you share the jenkins link of the test runs

Reply:
yes, i will raise a different pr for negative tests.

Reply:
Triggered the build pipeline with these parameters 
![image](https://github.com/user-attachments/assets/e4312f43-a97d-4ad9-a35e-6b9aa4cac348)

Reply:
@chandrams This is the jenkins build link. Can u please verify
 https://ci.app-svc-perf.corp.redhat.com/job/ExternalTeams/job/Autotune/job/kruize_functional_tests/276/